keps/sig-network/784-kube-proxy-component-config/README.md

```diff
@@ -127,6 +135,55 @@ The mitigations to those risks:
+| Conntrack              | Linux.Conntrack     | moved from root(generic) to linux (platform-specific) section    |
+| OOMScoreAdj            | Linux.OOMScoreAdj   | moved from root(generic) to linux (platform-specific) section    |
+| IPTables.MasqueradeAll | Linux.MasqueradeAll | moved from iptables (backend-specific) to root (generic) section |
+| NFTables.MasqueradeAll | Linux.MasqueradeAll | moved from nftables (backend-specific) to root (generic) section |
+| IPTables.SyncPeriod    | SyncPeriod          | moved from iptables (backend-specific) to root (generic) section |
+| NFTables.SyncPeriod    | SyncPeriod          | moved from nftables (backend-specific) to root (generic) section |
+| IPVS.SyncPeriod        | SyncPeriod          | moved from ipvs (backend-specific) to to root (generic) section  |
+| IPTables.MinSyncPeriod | MinSyncPeriod       | moved from iptables (backend-specific) to root (generic) section |
+| NFTables.MinSyncPeriod | MinSyncPeriod       | moved from nftables (backend-specific) to root (generic) section |
+| IPVS.MinSyncPeriod     | MinSyncPeriod       | moved from ipvs (backend-specific) to root (generic) section     |
+
+### Following fields will be changed
+| v1alpha1           | v1alpha2                 | DataType     | Comments                                                                                                       |
+|--------------------|--------------------------|--------------|----------------------------------------------------------------------------------------------------------------|
+| ClusterCIDR        | DetectLocal.ClusterCIDRs | list[string] | list of CIDR ranges for detecting local traffic                                                                |
+| BindAddress        | NodeIPOverride           | <same>       | no change in data type                                                                                         |
+| MetricsBindAddress | MetricsBindAddresses     | list[string] | list of CIDR ranges that contain valid node IPs to expose metrics server, instead of host port(ip:port) format |
+| HealthzBindAddress | HealthzBindAddresses     | list[string] | list of CIDR ranges that contain valid node IPs to expose healthz server, instead of host port(ip:port) format |                                      |
+
+### Following fields will be added
+| Field              | DataType         | Default Value    | Comments                                                                       |
+|--------------------|------------------|------------------|--------------------------------------------------------------------------------|
+| IPVS.MasqueradeBit | integer (32-bit) | 14               | IPVS will use this field instead of IPTables.MasqueradeBit                     |
+| Windows.Service    | boolean          | no-default-value | new field for existing --windows-service command line flag                     |
```

Comment:
Maybe `RunAsService`? (That seems to be a correct description: https://github.com/kubernetes/kubernetes/pull/60144)

Also, a boolean can't really not have a default value. That should be "false".